flytekit/bin/entrypoint.py

```diff
@@ -549,7 +549,13 @@ def fast_execute_task_cmd(additional_distribution: str, dest_dir: str, task_exec
 
     # Use the commandline to run the task execute command rather than calling it directly in python code
     # since the current runtime bytecode references the older user code, rather than the downloaded distribution.
-    p = subprocess.Popen(cmd)
+    env = os.environ.copy()
+    if all(os.path.realpath(path) != dest_dir for path in sys.path) and dest_dir is not None:
+        if "PYTHONPATH" in env:
+            env["PYTHONPATH"] += os.pathsep + dest_dir
```

Comment:
Can we resolve `dest_dir`?

```suggestion
            env["PYTHONPATH"] += os.pathsep + os.path.realpath(dest_dir)
```

Reply:
yeah. i think it's better to resolve it. Thanks!